src/app/staticanalysis/modules/staticAnalysisCommon.js

```diff
@@ -26,6 +42,17 @@ var basicFunctionTypes = {
   DELEGATECALL: buildFunctionSignature([], [basicTypes.BOOL], false)
 }
 
+var builtinFunctions = {
+  'keccak256()': true,
+  'sha3()': true,
+  'sha256()': true,
+  'ripemd160()': true,
+  'ecrecover(bytes32,uint8,bytes32,bytes32)': true,
+  'addmod(uint256,uint256,uint256)': true,
+  'mulmod(uint256,uint256,uint256)': true,
+  'selfdestruct(address)': true
```

Comment:
At the time i wrote this the compiler did not support revert or assert. But i will add it. Whats the complete signature? revert() i guess?